Exercise_1.py

```diff
@@ -1,22 +1,30 @@
-# Python code to implement iterative Binary  
-# Search. 
+# Python code to implement iterative Binary
+# Search.
   
-# It returns location of x in given array arr  
-# if present, else returns -1 
-def binarySearch(arr, l, r, x): 
+# It returns location of x in given array arr
+# if present, else returns -1
+def binarySearch(arr, l, r, x):
   
   #write your code here
+    if r>=1:
+        mid=l+(r-1)//2
```

Comment:
Why we don't do mid = (l+r)/2

Reply:
We can use (r+l)//2, cannot use (r+l)/2 as list expects indices to be integer and 'x/2' gives float.